src/Features/Core/Portable/InlineHints/AbstractInlineHintsService.cs

```diff
@@ -14,18 +15,18 @@ namespace Microsoft.CodeAnalysis.InlineHints
     internal abstract class AbstractInlineHintsService : IInlineHintsService
     {
         public async Task<ImmutableArray<InlineHint>> GetInlineHintsAsync(
-            Document document, TextSpan textSpan, CancellationToken cancellationToken)
+            Document document, TextSpan textSpan, InlineHintsOptions options, CancellationToken cancellationToken)
         {
             var inlineParameterService = document.GetLanguageService<IInlineParameterNameHintsService>();
             var inlineTypeService = document.GetLanguageService<IInlineTypeHintsService>();
 
             var parameters = inlineParameterService == null
                 ? ImmutableArray<InlineHint>.Empty
-                : await inlineParameterService.GetInlineHintsAsync(document, textSpan, cancellationToken).ConfigureAwait(false);
+                : await inlineParameterService.GetInlineHintsAsync(document, textSpan, options.ParameterOptions, options.DisplayOptions, cancellationToken).ConfigureAwait(false);
 
             var types = inlineTypeService == null
                 ? ImmutableArray<InlineHint>.Empty
-                : await inlineTypeService.GetInlineHintsAsync(document, textSpan, cancellationToken).ConfigureAwait(false);
+                : await inlineTypeService.GetInlineHintsAsync(document, textSpan, options.TypeOptions, options.DisplayOptions, cancellationToken).ConfigureAwait(false);
```

Comment:
i don't love that it's called .DisplayOptions but returns XXXDecriptionOPtions.  Display vs Description should ideally be aligned plz.

Reply:
Yeah, DescriptionOptions sounds odd though - not sure what would "inline hint description" mean. Display options seemed better even if it doesn't match the type name.